src/bash/Makefile

```diff
@@ -9,8 +9,22 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 
 	dget -u https://launchpad.net/debian/+archive/primary/+sourcefiles/bash/$(BASH_VERSION_FULL)/bash_$(BASH_VERSION_FULL).dsc
 
+	# Apply plugin suport patches
+	quilt push -a
+
 	pushd bash-$(BASH_VERSION_MAJOR)
+
+	# build package
 	DEB_BUILD_OPTIONS=nocheck dpkg-buildpackage -us -uc -b -j$(SONIC_CONFIG_MAKE_JOBS) --admindir $(SONIC_DPKG_ADMINDIR)
+
+	# copy UT code.
+	cp -a ../Files/. ./
+
+	# run UT after build package because we need config.h
+	pushd unittest
+	make
```

Comment:
>make [](http://example.com/codeflow?start=1&length=4)

you may `make && make test`

Reply:
you can `make -C` so no need change directory.

Reply:
Fixed.